backend/pkg/api/query.go

```diff
@@ -72,25 +68,33 @@ func NewQueryResponse(response *http.Response) (*QueryResponse, error) {
 
 	body, err := ioutil.ReadAll(response.Body)
 	if err != nil {
-		log.DefaultLogger.Error(err.Error())
+		log.DefaultLogger.Error("NewQueryResponse: ioutil.ReadAll: " + err.Error())
 		return nil, err
 	}
 
 	var qr QueryResponse
 	err = json.Unmarshal(body, &qr)
-
 	if err != nil {
-		log.DefaultLogger.Error(err.Error())
+		log.DefaultLogger.Error("NewQueryResponse: json.Unmarshal: " + err.Error())
 		return nil, err
 	}
 
 	return &qr, nil
 }
 
 func (qr *QueryResponse) Rows() [][]interface{} {
-	return qr.Results.A.Tables[0].Rows
+
```

Comment:
Nit: just for readability, might be helpful to add some other helper methods, maybe something like:

```
func (qr *QueryResponse) Tables() []interface{} {
  if len(qr.Results.A.Tables) > 0 {
    return qr.Results.A.Tables[0]
  }
}
```

Not sure about the return type! 

Could make `Rows()` and `Columns()` a bit tidier, but could be personal pref, so up to you!

Reply:
Yup it's pretty messy